acceptance/tests/resource/file/source_attribute.rb

```diff
@@ -164,7 +164,7 @@ class source_test_module {
 
     step "second run should not update file"
```

Comment:
I may not be grokking the problem set here, but to make this test resilient to timing issues before this step we should should get the time on the master when we created the files and on the agent after the first application, then ensure that the time on the agent is after the time when we created the files on the master before running this step.